javascript/apps/taiga/src/app/modules/project/feature-kanban/components/workflow/kanban-workflow.component.css

```diff
@@ -39,3 +45,11 @@ tui-scrollbar {
 tg-kanban-story {
   margin: 0;
 }
+
+tg-ui-drag-in-progress tg-kanban-status {
+  block-size: var(--drag-status-height);
+  box-shadow: 3px 4px 14px 0 rgba(0, 138, 168, 0.15);
+  cursor: grab;
+  display: block;
```

Comment:
When I do hover in the status title the cursor should be pointer
![image](https://github.com/taigaio/taiga/assets/5970959/3b83095f-90fa-472c-921e-f0ec635c52be)
(check the design 'status of status column')